workspaces/kener/plugins/kener-status/.changeset/eight-colts-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kener-status': minor
+---
+
+Refactor plugin structure according to review feedback PR#2790. Restoring .eslintrc.cjs.
```

Comment:
This doesn't need to be in a changeset, these are just part of the normal process for a PR as it's reviewed. Changesets are only needed for the specific change this PR is for.